crates/protocol/src/operator/mod.rs

```diff
@@ -114,9 +114,10 @@ struct PermissionParseError {
 }
 
 // Serialization
+pub const SIGNING_PREFIX: &[u8] = b"WARG-OPERATOR-RECORD-SIGNATURE-V0";
```

Comment:
Is this to allow signing with another crypto implementation? I'd rather the interface be "the entire thing to be signed" than reimplement the slightly-subtle payload encoding.

Reply:
This was for the "Componentizing" implementation, exposing the signing prefix outside the crate so that it is possible for the signing to happen outside of the Wasm binary. We could it as you suggest but it would involve more copying and allocations.

I'm happy to pull out these changes for the "Componentizing" implementations into a separate PR. That might make more sense. So that they can be evaluated for the full usage.

Reply:
These payloads are really quite small. Compared to the ECDSA signing you're doing immediately after, copying the payload out is going to be effectively free.